golf2/apps/dcrypto_test/Makefile

```
@@ -1,14 +1,17 @@
 CURRENT_DIR := $(dir $(abspath $(lastword $(MAKEFILE_LIST))))
 
 TOCK_ARCH ?= cortex-m3
-TOCK_USERLAND_BASE_DIR = $(CURRENT_DIR)../../../tock/userland
+TOCK_USERLAND_BASE_DIR = $(CURRENT_DIR)../../../../libtock-c
 BUILDDIR ?= $(CURRENT_DIR)/build/$(TOCK_ARCH)
 
 C_SRCS   := $(wildcard *.c)
 
 OBJS += $(patsubst %.c,$(BUILDDIR)/%.o,$(C_SRCS))
 
-CPPFLAGS += -DSTACK_SIZE=2048
+TOCK_APP_CONFIG = -Xlinker --defsym=STACK_SIZE=$$(STACK_SIZE)\
+                  -Xlinker --defsym=APP_HEAP_SIZE=$$(APP_HEAP_SIZE)\
+                   -Xlinker --defsym=KERNEL_HEAP_SIZE=$$(KERNEL_HEAP_SIZE)
```

Nitpick: Remove 1 space of indentation so these line up.

```suggestion
                  -Xlinker --defsym=KERNEL_HEAP_SIZE=$$(KERNEL_HEAP_SIZE)
```